.github/actions/javascript/reviewerChecklist/reviewerChecklist.js

```diff
@@ -76,6 +86,8 @@ function checkIssueForCompletedChecklist(numberOfChecklistItems) {
 }
 
 getNumberOfItemsFromReviewerChecklist()
-    .then(checkIssueForCompletedChecklist, (err) => {
+    .then(checkIssueForCompletedChecklist)
```

Comment:
Should we implement this same change in the author checklist too?

Reply:
Can't hurt!